tests/CI/run_docker_setup.sh

```diff
@@ -150,20 +150,23 @@ prepareEnvironment() {
     done
   fi
 
-  echo "Generated config file is:"
-  cat "${SERVERCONFIG}"
   cp "${SERVERCONFIG}" "${CLIENTCONFIG}"
 
-  if [[ -n "${SERVER_USE_M2CRYPTO+x}" ]]; then
-    echo "export DIRAC_USE_M2CRYPTO=${SERVER_USE_M2CRYPTO}" >> "${SERVERCONFIG}"
-    echo "export DIRAC_M2CRYPTO_SPLIT_HANDSHAKE=Yes" >> "${SERVERCONFIG}"
+  if [[ "${SERVER_USE_PYTHON3:-}" == "Yes" ]]; then
+    echo "INSTALLOPTIONS+=(\"--pythonVersion=3\")" >> "${SERVERCONFIG}"
   fi
 
-  if [[ -n "${CLIENT_USE_M2CRYPTO+x}" ]]; then
-    echo "export DIRAC_USE_M2CRYPTO=${CLIENT_USE_M2CRYPTO}" >> "${CLIENTCONFIG}"
-    echo "export DIRAC_M2CRYPTO_SPLIT_HANDSHAKE=Yes" >> "${CLIENTCONFIG}"
+  echo "${CLIENT_USE_PYTHON3}"
+  if [[ "${CLIENT_USE_PYTHON3:-}" == "Yes" ]]; then
+    echo "INSTALLOPTIONS+=(\"--pythonVersion=3\")" >> "${CLIENTCONFIG}"
```

Comment:
We just take the last version?

Reply:
For now DIRACOS2 is using Python 3.8 but I don't think this option should be used to pick the Python version. Instead the full Python version will be implicit from the DIRACOS version.